sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -139,9 +140,9 @@ class JobOperations(_ScopeDependentOperations):
     :type operation_scope: ~azure.ai.ml._scope_dependent_operations.OperationScope
     :param operation_config: Common configuration for operations classes of an MLClient object.
     :type operation_config: ~azure.ai.ml._scope_dependent_operations.OperationConfig
-    :param service_client_02_2023_preview: Service client to allow end users to operate on Azure Machine Learning
+    :param service_client: Service client to allow end users to operate on Azure Machine Learning
```

Comment:
Just curious, as this is a public interface, why the public interface review not detect this?

https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.operations.joboperations?view=azure-python

Reply:
Has been detected:
![Uploading image.png…]()